docs/atomic-storage.1.md

```diff
@@ -90,5 +90,15 @@ Drivers supported: devicemapper, overlay, overlay2
 **--vgroup**
 The name of the volume group for the storage pool.
 
+# reset OPTIONS
+**-h** **--help**
+  Print usage statement
+
+**--graph**
+Root of the docker runtime. If you are running docker at the default
+location (/var/lib/docker), you don't need to pass this flag. However
+if you are running docker at a custom location. This flag must be set.
+
+
```

Comment:
We probably need to specify default behavior if --graph is not specified. `By default /var/lib/docker or /var/lib/docker-latest/ will be reset. If both are present or none are present user needs to specify the path using --graph option.`